README.md

```diff
@@ -1,3 +1,24 @@
-# Superset
+# Readme
 
 [Apache Superset](https://superset.apache.org/) Docker image with additional libraries used with Medic's CHT deployments. This follows Superset's [advice about using a custom image](https://superset.apache.org/docs/installation/docker-builds/#building-your-own-production-docker-image).
+
+# Docker Image
+This docker image currently comes with following packages installed that are used in generally most of the superset deployments medic is supporting. 
+
+- `psycopg2-binary`: To connect to postgres databases
+- `Authlib`: To support OAuth login
+- `openpyxl`: To support uploading excel files for analysis
+- `Pillow`: For Alert and Report
+- `playwright`: For taking screenshots for Alerts & Reports
+
+## Usage
+To use this version of superset on your deployments, update your `values.yaml` to point to this superset image repo instead of `apachesuperset.docker.scarf.sh/apache/superset`
+
+```yaml
+image:
+  repository: public.ecr.aws/medic/superset
+  tag: "5.0.0"
```

Comment:
I see a 5.0.0 tag [in ECR](https://gallery.ecr.aws/medic/superset) but I don't see any versions [in GH](https://github.com/medic/superset) - should there be on in GH?

Reply:
We will keep the same versioning as apache superset, I am not sure if creating release on GH repo would be helpful.